pkg/driver/controller.go

```diff
@@ -622,15 +622,16 @@ func (d *controllerService) CreateSnapshot(ctx context.Context, req *csi.CreateS
 		return nil, status.Errorf(codes.InvalidArgument, "Invalid tag value: %v", err)
 	}
 
-	for k, v := range addTags {
-		snapshotTags[k] = v
-	}
-
 	if d.driverOptions.kubernetesClusterID != "" {
 		resourceLifecycleTag := ResourceLifecycleTagPrefix + d.driverOptions.kubernetesClusterID
 		snapshotTags[resourceLifecycleTag] = ResourceLifecycleOwned
 		snapshotTags[NameTag] = d.driverOptions.kubernetesClusterID + "-dynamic-" + snapshotName
 	}
+
+	for k, v := range addTags {
+		snapshotTags[k] = v
+	}
+
```

Comment:
Can we move this after the for loop on line 634-637 instead, for consistency with how it works in `CreateVolume`: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/5d75bf4be309d3b554d3959a79cc8f05ed65d4d2/pkg/driver/controller.go#L251-L266

Reply:
Sure